lms/static/sass/features/_course-upgrade-message.scss

```diff
@@ -3,11 +3,10 @@
   the creative property. This will allow us to better track individual performance of each style of the message.
   Search for the courseware_verified_certificate_upsell promotion ID.
  */
-$upgrade-message-background-color: $blue-d1;
 
 // Expanded upgrade message
 .vc-message {
-  background: $blue-d1;
+  background: $lms-hero-color;
```

Comment:
Not necessary here, but what do you think we should do with some of the more obscure colors? Should we keep them in theme-colors?

Reply:
Good question, and I don't have a good answer for you. Let's discuss.